subprojects/docs/src/docs/userguide/migration/upgrading_version_6.adoc

```diff
@@ -45,6 +45,7 @@ The API supporting the Java Toolchain feature in `org.gradle.jvm.toolchain` is n
 ==== Updates to default tool integration versions
 
 - JaCoCo has been updated to http://www.jacoco.org/jacoco/trunk/doc/changes.html[0.8.6].
+- CodeNarc has been updated to https://github.com/CodeNarc/CodeNarc/blob/v1.6.1/CHANGELOG.md[CodeNarc 2.0.0].
```

Comment:
Don't you mean? https://github.com/CodeNarc/CodeNarc/blob/v2.0.0/CHANGELOG.md#version-200----oct-2020